src/GPIOMessaging.toit

```diff
@@ -1,75 +0,0 @@
-import serialization
-
-///Small class containing properties regarding setting a specific value for a specific GPIO pin.
-///If the property resetTime is set to anything else than null, the original value should be changed back after the reset time has been hit.
-class GPIOTriggerMessage:
-  constructor.initAll .pin .value .resetTime:
-  constructor .pin .value:
-  pin /string //Which pin this should invoke
-  value /int //What value the pin should have
-  resetTime /int? := null   //Time before the pin should be reset to originial value. Null if never
-
-///Validator for GPIOInvokeMessage. Ensures that the received message can be processed 
-class GPIOTriggerMessageValidator:
-
-  //Validates the given GPIOTriggerMessage.
-  //True if required properties are correct set, false otherwise
-  static IsObjectValid --input /GPIOTriggerMessage ->bool:
```

Comment:
I removed this function, as the fields in the message are non-nullable (they don't have a "?"). As such most of the checks were not necessary.

Reply:
All functions in Toit are lower case.